drivers/usb/usbstor_new/disk.c

```diff
@@ -329,7 +395,7 @@ USBSTOR_HandleQueryProperty(
         //
         // get inquiry data
         //
-        InquiryData = (PUFI_INQUIRY_RESPONSE)PDODeviceExtension->InquiryData;
+        InquiryData = &PDODeviceExtension->InquiryData;
```

Comment:
This change makes no sense. `PDODeviceExtension->InquiryData` is a PVOID, and the structure does not contain inquiry data behind his.